### PR TITLE
Implement network == none

### PR DESCRIPTION
**Pull request by baude**
When network == none, the container should only have a
loopback interface and that's it.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

**Comment by mheon** [comment]
LGTM pending tests

**Comment by mheon** [comment]
Tests seem to be failing. Might be a bug in the code, but I recall a lot of tests that should have been failing returning 0 when I was working on the DB, so that could be related

File: test/podman_networking.bats [hunk 1]
```diff
@@ -11,13 +11,19 @@ function setup() {
 }
 
 @test "test network connection with default bridge" {
-    run ${KPOD_BINARY} ${KPOD_OPTIONS} run -dt ${ALPINE} wget www.yahoo.com
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} run -dt ${ALPINE} wget projectatomic.org
```
Comment (rhatdan) [inline test/podman_networking.bats]: No need for -dt.

File: test/podman_networking.bats [hunk 2]
```diff
@@ -11,13 +11,19 @@ function setup() {
...
     echo "$output"
     [ "$status" -eq 0 ]
 }
 
 @test "test network connection with host" {
-    run ${KPOD_BINARY} ${KPOD_OPTIONS} run -dt --network host ${ALPINE} wget www.yahoo.com
+    run ${KPOD_BINARY} ${KPOD_OPTIONS} run -dt --network host ${ALPINE} wget projectatomic.org
```
Comment (rhatdan) [inline test/podman_networking.bats]: no need for -dt

**Comment by baude** [comment]
bot, retest this please

**Comment by rhatdan** [comment]
@baude I think you travis issues look legitimate.

**Comment by rhatdan** [comment]
@baude needs a rebase.

**Comment by baude** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**